src/core_plugins/getting_started/public/lib/add_setup_work.js

```diff
@@ -11,6 +11,67 @@ import {
   CREATE_INDEX_PATTERN_ROUTE
 } from './constants';
 
+function handleExistingIndexPatternsScenario(indexPatterns, currentRoute, config) {
+  // If index patterns exist, we're not going to show the user the Getting Starte page.
+  // So we can show the chrome again at this point.
+  uiChrome.setVisible(true);
+
+  // The user need not see the Getting Started page, so opt them out of it
+  optOutOfGettingStarted();
+
+  // Some routes require a default index pattern to be present. If we're
+  // NOT on such a route, there's nothing more to do; send the user on their way
+  if (!currentRoute.requireDefaultIndex) {
+    return;
+  }
+
+  // Otherwise, check if we have a default index pattern
+  let defaultIndexPattern = config.get('defaultIndex');
```

Comment:
This defaultIndex logic doesn't seem appropriate in the getting started guide, as it doesn't seem to be related at all.  If someone were to disable the getting_started plugin entirely, it would break the default index behavior.